content/references/kata-trainer.md

```diff
@@ -3,41 +3,78 @@ title: Kata Trainer
 kind: reference
 ---
 
-:::note
-*This page is a stub, please consider contributing to it.*
-:::
+import ThemedImage from "@theme/ThemedImage";
```

Comment:
Please change the extension of this file to `.mdx` (`kata-trainer.mdx`) since we're using React components now.